frontend/src/pages/projects/screens/spawner/imageSelector/ImageVersionSelector.tsx

```diff
@@ -52,10 +59,43 @@ const ImageVersionSelector: React.FC<ImageVersionSelectorProps> = ({
       label: `${imageStream.metadata.name}-${imageVersion.name}`,
       dropdownLabel: (
         <ImageVersionTooltip dependencies={getImageVersionDependencies(imageVersion, false)}>
-          {optionObject.toString()}
+          <Flex>
+            <FlexItem>
+              {`${optionObject.imageVersion.name} ${
+                optionObject.imageVersion.annotations?.['opendatahub.io/notebook-build-commit']
+                  ? `(${optionObject.imageVersion.annotations['opendatahub.io/notebook-build-commit']})`
+                  : ''
+              }`}
+            </FlexItem>
+            <FlexItem align={{ default: 'alignRight' }}>
+              {optionObject.imageVersion.annotations?.[
+                'opendatahub.io/workbench-image-recommended'
+              ] === 'true' && (
+                <Label
+                  data-testid="notebook-image-availability"
+                  isCompact
+                  color="green"
+                  icon={<CheckCircleIcon />}
+                >
+                  Latest
+                </Label>
+              )}
+            </FlexItem>
+          </Flex>
         </ImageVersionTooltip>
       ),
-      description: getImageVersionSoftwareString(imageVersion),
+      description: (
+        <div>
+          <div data-testid="workbench-image-version-software">
+            <b>Software:</b> {getImageVersionSoftwareString(imageVersion)}
+          </div>
+          {imageBuildDate && (
+            <div data-testid="workbench-image-version-build-date">
+              <b>Build date:</b> {convertISOTimeToHumanReadable(imageBuildDate)}
```

Comment:
please just use the timestamp component from patternfly for consistency. the formatting may be slightly different, thats ok